superbench/runner/runner.py

```diff
@@ -48,7 +48,7 @@ def __set_logger(self, filename):
         """
         SuperBenchLogger.add_handler(logger.logger, filename=str(self._output_path / filename))
 
-    def __validate_sb_config(self):
+    def __validate_sb_config(self):    # noqa: C901
```

Comment:
Why ignore flake8 warning?

Reply:
function too complex, will separate validation part in the future

Reply:
Shall we fix it this time?